src/main/java/racingcar/RacingSetting.java

```diff
@@ -0,0 +1,46 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingSetting {
+    public List<Car> cars = new ArrayList<>();
+
+    private void createCarNames() {
```

Comment:
"차 이름을 생성한다"라는 메서드 명은 적절해보이진 않은것같아요. 현재 Car객체를 생성하고 있으므로 createCar()라는 메서드 명이 나아보이네요! 
```suggestion
    private void createCar() {
```